Make registering proxy mandatory for katello (#3856)
theforeman/foreman-documentation: guides/common/modules/proc_registering-capsule-to-satellite-server.adoc

[id="Registering_Proxy_to_Server_{context}"]
= Registering to {ProjectServer}

> Registering the base operating system on which you want to install {SmartProxyServer} to {ProjectServer} enables you to download the installation packages from your synchronized repositories.

Reviewer: Style wise it feels like a long sentence. I'm not sure we need to mention `"base operating system". I'd be tempted to say 
```suggestion
Registering the host for your {SmartProxyServer} to {ProjectServer} enables you to download the installation packages from your synchronized repositories.
```

Reviewer 2: If we use it elsewhere too, then I would also stick to it.

= Registering to {ProjectServer}

Use this procedure to register the base operating system on which you want to install {SmartProxyServer} to {ProjectServer}.
ifdef::katello[]
Registering your {SmartProxyServer} as a content host enables you to download the installation packages from your synced repositories.

> endif::[]

Author: Rewrote, as one sentence.
